### DISCUSSION TITLE
Feature request: Sleep mode / night time listening

### DISCUSSION THREAD

**Discussion by littlesandra88** [PaulWoitaschek/Voice#2266] [Ideas]
Dear all =)

**Problem**
Often I listen to a book when I am in bed before I go to sleep. And often I fall asleep while the book is playing. In the morning the book is completed, and I have no idea where in the book I fell asleep.

**Suggestion**
A sleep mode / night time listening mode, where one can enter a time duration before the book stops playing or a number of chapters before it stop playing.

Some books are one file and other one file for each chapter. So there could be some user confusion if someone selects "stop after 1 chapter" in a one file book with multiple chapters, as the entire book will be seen as one chapter.

Hugs,
Sandra =)

**Comment by PaulWoitaschek** [comment]
Hi Sandra, try clicking on the little moon in the top of the playback screen :)

**Reply by littlesandra88** [reply]
Wow! That is unbelievable! =)

Thanks =)
